### PR TITLE
update README with setup on OCP

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,9 +34,54 @@ To delete the kind environment, use `make clean`.
...
     ```
-    kubectl apply -k deploy/config/manifests
+
+5. Accept the `ManagedClusters` on the hub cluster.
+   
+   ```
+   $ oc patch managedclusters <managedcluster name> --type merge --patch '{"spec":{"hubAcceptsClient":true}}'
+   ```
+   
+Setup Addon:
```
Comment (skeeey) [inline README.md:59]: On the hub cluster, setup addon?

File: README.md [hunk 2]
```diff
@@ -34,9 +34,54 @@ To delete the kind environment, use `make clean`.
...
+
+1. Apply the manifests of submariner-addon.
+
     ```
+    $ oc apply -k deploy/config/manifests
+    ```
+
+Setup Submariner:
```
Comment (skeeey) [inline README.md:67]: On the hub cluster, setup submariner?

File: README.md (hunk 1 above)
Comment (zhiweiyin318) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (zhiweiyin318) [reply README.md]: done

**Pull request merged**